adapter/src/dbt/adapters/fal_experimental/utils.py

```diff
@@ -6,12 +6,16 @@
 
 import dbt.exceptions
 from fal.packages.environments.base import (
-    BaseEnvironment,
     BasicCallable,
     EnvironmentConnection,
 )
```

Comment:
Should we remove these imports now?

Reply:
In general I thought when we introduced isolate we could remove the fal dependency